.prettierignore

```diff
@@ -0,0 +1,13 @@
+# exclude everything by default
+*.*
+# exclude these files
+Dockerfile
+# exclude these directories
+/bin/
+/dist/
+/docker/
+/node_modules/
+# white-list files we want to process
+!*.js
+!*.md
+!*.scss
```

Comment:
I know this is separate from the TS PR, but should this also white-list .ts and .tsx files?

Reply:
yep, we should. I'll add that before merging if the TS PR gets merged